.vitepress/config.ts

```diff
@@ -272,14 +272,18 @@ export const sidebarsExamples = (): DefaultTheme.SidebarItem[] => [
         text: 'Zod OpenAPI',
         link: '/examples/zod-openapi',
       },
-      {
-        text: 'Swagger UI',
-        link: '/examples/swagger-ui',
-      },
       {
         text: 'Hono OpenAPI',
         link: '/examples/hono-openapi',
       },
+      {
+        text: 'Scalar',
```

Comment:
Can you put the `Scalar` under `Swagger UI`? Since `Swagger UI` was created before `Scalar`, I would like to place it above.

Reply:
Sure. I had them ordered by quality – Scalar is a vastly superior product, rapidly gaining adoption among companies including the likes of Microsoft. That being said, I moved it below Swagger UI.

Reply:
Thank you for understanding.